soc/arm/atmel_sam/sam4e/Kconfig.series

```diff
@@ -2,14 +2,18 @@
 
 # Copyright (c) 2017 Justin Watson
 # Copyright (c) 2018 Vincent van der Locht
-# Copyright (c) 2019 Gerson Fernando Budke
+# Copyright (c) 2019-2020 Gerson Fernando Budke <nandojve@gmail.com>
 # SPDX-License-Identifier: Apache-2.0
 
 config SOC_SERIES_SAM4E
 	bool "Atmel SAM4E MCU"
 	select ARM
+	select ARM_MPU
```

Comment:
`CPU_HAS_ARM_MPU` is the symbol that indicates that ARM MPU is available on an SoC, and`ARM_MPU` is the symbol used to enable it.

Unless it is a requirement that you enable the ARM MPU for this SoC, `ARM_MPU` should be selected by the board using this SoC.

Reply:
Done!